lib/phonebook/ui/pages/association_editor_page/association_editor_page.dart

```diff
@@ -293,7 +293,28 @@ class AssociationEditorPage extends HookConsumerWidget {
               builder: (context, associationMembers) =>
                   associationMembers.isEmpty
                       ? const Text(PhonebookTextConstants.noMember)
-                      : Column(
+                      : ReorderableListView(
+                          physics: const BouncingScrollPhysics(),
+                          proxyDecorator: (child, index, animation) {
+                            return Material(
+                              child: FadeTransition(
+                                opacity: animation,
+                                child: child,
+                              ),
+                            );
+                          },
+                          onReorder: (int oldIndex, int newIndex) {
+                            associationMemberListNotifier.updateMember(
+                              associationMemberSortedList[oldIndex],
```

Comment:
When reordering, shouldn't at least two memberships change swap their order ?

Reply:
Should be fixed with the last commit